MzML/Mzml.cs

```diff
@@ -354,6 +354,24 @@ private static MsDataScan GetMsDataOneBasedScanFromConnection(Generated.mzMLType
                 }
             }
 
+            double high = double.NaN;
+            double low = double.NaN;
+
+            if (_mzMLConnection.run.spectrumList.spectrum[oneBasedIndex - 1].scanList.scan[0].scanWindowList != null)
```

Comment:
set this connection to a variable so it doesn't need to be looked up again for the foreach

Reply:
done